### ISSUE TITLE
keras version match tensorflow

### ISSUE THREAD

**Issue by hanxuel** [google/mipnerf#13]
Hi, when I am trying to reproduce the result, I installed on cuda10.1 py3.6 tensorflow2.3.1, keras2.7.0

Then I met the issue when I import tensorflow, and from keras import optimizers,

"another metric with the same name already exists".

I would like to know what's the version you use during your experiments?

Thanks in advance!

**Comment by hanxuel** [comment]
solved by changing keras version to 2.6.*

**Comment by AndyYuan96** [comment]
> 

Hi, Hanxue, can I get your email, as I have some question after reading your paper GCC-3D, and I don't find your email in the paper. thank you very much.

**Comment by AndyYuan96** [comment]
> Hi, when I am trying to reproduce the result, I installed on cuda10.1 py3.6 tensorflow2.3.1, keras2.7.0
> 
> Then I met the issue when I import tensorflow, and from keras import optimizers,
> 
> "another metric with the same name already exists".
> 
> I would like to know what's the version you use during your experiments?
> 
> Thanks in advance!

Hi, Hanxue, can I get your email, as I have some question after reading your paper GCC-3D, and I don't find your email in the paper. thank you very much.

**Comment by hanxuel** [comment]
> > Hi, when I am trying to reproduce the result, I installed on cuda10.1 py3.6 tensorflow2.3.1, keras2.7.0
> > Then I met the issue when I import tensorflow, and from keras import optimizers,
> > "another metric with the same name already exists".
> > I would like to know what's the version you use during your experiments?
> > Thanks in advance!
> 
> Hi, Hanxue, can I get your email, as I have some question after reading your paper GCC-3D, and I don't find your email in the paper. thank you very much.

Hi, sorry for this inconvenience. my email is haliang@student.ethz.ch
